docs/source/tutorials/describing_tasks.ipynb

```diff
@@ -1,20 +1,563 @@
 {
```

Comment:
<p>Should we add a note here explaining how those properties can be used programmatically when running a script instead of an interactive session in a notebook? While editing it I was left wondering that myself. Basically, explain to the users how this can be useful in a real-life situation?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2879/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I am not sure I understand what you are aiming at.  <code>how those properties can be used programmatically when running a script</code> - how would that differ from running in a notebook in this context? </p>

Reply:
A user has to write an application. How is he going to use those properties besides the toy example we offer? Why should they care? In what real-life context should they use those properties?

Reply:
You are saying the examples should be a more realistic reflection of a use case?  I guess in that case we should pick a use case and use that throughout the tutorials - is that what you are aiming at?  

The comment is specifically at the section about obtaining results and error information from completed tasks.  I don't think there is much motivation needed why a user would care about that information: why else would they be running the task?  So I guess your comments aims at the examples in general across the tutorials maybe, is that correct?

Reply:
> You are saying the examples should be a more realistic reflection of a use case?

No.

> I guess in that case we should pick a use case and use that throughout the tutorials - is that what you are aiming at?

No.

> The comment is specifically at the section about obtaining results and error information from completed tasks. I don't think there is much motivation needed why a user would care about that information: why else would they be running the task?

I do not understand your question. I try one last time then I will give up. In what situation a user would want to use the two task properties you illustrate? How would they be useful when running a scientific application? How would be introspection useful when a domain scientist runs (or is writing?) an application with RADICAL-Pilot?

> So I guess your comments aims at the examples in general across the tutorials maybe, is that correct?

No.

Reply:
I gave some examples on how the attributes can be used.